transport.go

```diff
@@ -127,13 +127,23 @@ func transactionEnvelopeFromBody(event *Event, dsn *Dsn, sentAt time.Time, body
 		return nil, err
 	}
 
+	var itemType string
+	switch event.Type {
+	case transactionType:
+		itemType = transactionType
+	default:
+		itemType = eventType
+	}
+
 	// Item header
 	err = enc.Encode(struct {
-		Type   string `json:"type"`
-		Length int    `json:"length"`
+		Type        string `json:"type"`
+		Length      int    `json:"length"`
+		ContentType string `json:"content_type"`
 	}{
-		Type:   transactionType,
-		Length: len(body),
+		Type:        itemType,
+		Length:      len(body),
+		ContentType: "application/json",
```

Comment:
The ContentType part is new, is it obligatory now?

Reply:
Not required, same as `length`. But as pointed out in most examples on https://develop.sentry.dev/sdk/envelopes/, I opted to include it.

Reply:
Interestingly, the docs only cover the field in the context of attachments:

```
content_type
    String, optional. The content type of the attachment payload. Any [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml) may be used; the default is application/octet-stream.
```

Haven't checked, but we might be ignoring the content-type field for non-attachment payloads.

Reply:
I opted to remove it again.